source/payment_flow/index.html.md.erb

```diff
@@ -120,6 +120,11 @@ they have completed their payment attempt on GOV.UK Pay. This should not be
 JSON-encoded, because backslashes are invalid characters. With test accounts,
 you can [use HTTP for return URLs instead of HTTPS](/security/#https).
 
+You can [prefill the fields on the payment page](/optional_features/prefill_user_details) with the user's details if you collected that information:
+
+- earlier in your service, before you redirected your user to GOV.UK Pay
```

Comment:
I think it should say something along the lines 'before you created the charge'. When the user is redirected the charge was already created thus it's too late to include/exclude the billing address

Reply:
As discussed, I've tweaked to just "earlier in the service" as this should be clear on its own.